CHANGES.rst

```diff
@@ -5,7 +5,15 @@ Changelog of lizard-auth-client
 1.14 (unreleased)
 -----------------
 
-- Nothing changed yet.
+- Put the attemp_auto_login function into a middlware.
```

Comment:
Misschien integreren in onderstaande change entry? "Also available as middleware"?